src/api.v2/controllers/cellLevelMetaController.js

```diff
@@ -5,8 +5,11 @@ const sqlClient = require('../../sql/sqlClient');
 const CellLevelMeta = require('../model/CellLevelMeta');
 const CellLevelMetaToExperiment = require('../model/CellLevelMetaToExperiment');
 const { getFileUploadUrls } = require('../helpers/s3/signedUrl');
+const getLogger = require('../../utils/getLogger');
 const OK = require('../../utils/responses/OK');
 
+const logger = getLogger('[CellLevelController] - ');
```

Comment:
use `CellLevelMetaController`

Reply:
Ideally either use everywhere `CellLevelMeta` or `CellLevelMetadata`, for consistency